### DISCUSSION TITLE
Looking for MQTT Published message examples

### DISCUSSION THREAD

**Discussion by Bugman1400** [aqualinkd/AqualinkD#318] [Q&A]
Just trying to turn my Pool Light ON/OFF with MQTT. My non-supported home automation software (Homeseer HS4 PRO) has a MQTT interface and all payload messages seemed to get received. I am looking for message syntax examples for sending a '1' or '0' to AqualinkD to control the Pool Light. I believe the messages are in a RAW format.
I also have access to MQTT Explorer.

**Comment by sfeakes** [comment]
it really is as simply as sending 1 or 0 to the topic.  Using command line (if you have mosquito_pub installed) would look something like
```
mosquitto_pub -h <hostname> -t aqualinkd/<Aux_3>/set -m 1
```
Change `<hostname>` and `<Aux_3>` to you system

As for how you configure a switch in Homeseer to send the 1, (or 0) you would have to look that up. The below seems to have examples of what you are after.
https://forums.homeseer.com/forum/homeseer-products-services/general-discussion-area/99336-homeseer-mqtt-plugin/page5

If you install something like MQTT explorer on your desktop machine, that will also really help you see the messages that are sent / received.  Plus you can sent your own easily to test things.
Below is an example, if I hit "Publish" ( notice I'm sending `1` to the topic `aqualinkd/Aux_3/set` ) that would turn on my Pool Light.
![Screenshot 2024-06-27 at 9 15 35 PM](https://github.com/sfeakes/AqualinkD/assets/14110299/1e7443dc-e88b-44a3-8c82-bf86c3f4bc0e)

**Reply by Bugman1400** [reply]
crap....this was rather easy. I was missing the SET command in the Publish message. Seems to work now.
Mucho gracias.
